ansible/playbooks/reset_morio.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Reset Morio configuration on the EC2 instances
+  hosts: all
+  become: yes
+  become_method: sudo
+  tasks:
+    - name: Make backup directory
+      shell: mkdir -p ~/.moriod
+      ignore_errors: true
+
+    - name: Backup version info
+      shell: cp /etc/morio/moriod/version* ~/.moriod/
```

Comment:
As a rule of thumb, we should only use `shell` in Ansible when nothing more suitable is available.

In this case, we can use the built-in file module: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html

Reply:
I've rewrote it like this.

ansible.builtin.copy:
  src: /etc/morio/moriod/version*
  dest: "{{ ansible_env.HOME }}/.moriod/"
  remote_src: yes